docs/docs/userguide/PREREQUISITES.md

```diff
@@ -2,46 +2,42 @@
 
-## Requirements
-
-In order to deploy Atlassian’s Data Center infrastructure to Amazon Web Services (AWS), the following are required:
-
-1. An understanding of [Kubernetes](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/){.external} and [Helm](https://helm.sh/){.external} concepts.
-2. An understanding of [Terraform](https://www.terraform.io/){.external}.
-3. An AWS account with admin access. 
-
 ## Environment setup
 
-Before creating the infrastructure, make sure that your development environment is configured with the following tools:
+Its advised that the tooling below is installed to your development environment. An understanding of these tools and their associated concepts is also advisable.
 
 1. [Terraform](#terraform) 
 2. [Helm v3.3 or later](#helm)
 3. [AWS CLI](#aws-cli)
-4. [Kubernetes cluster monitoring tools](#kubernetes-tools) (optional)
+4. [Kubectl](#kubectl) (optional)
+5. [Kubernetes cluster monitoring tools](#kubernetes-cluster-monitoring-tools) (optional)
 
 ### :material-terraform: Terraform
 
 Terraform is an open-source infrastructure as code tool that provides a consistent CLI workflow to create and manage the infrastructure of cloud environments. 
 
 This project uses Terraform to create and manage the Atlassian Data Center infrastructure on AWS for use with supported Data Center products. 
 
-!!! info "Currently, not all Data Center products are supported." 
-    At this stage, Bamboo Data Center is the only supported product. 
+!!! warning "Supported Products and Platforms"  
+
+    * [AWS](https://aws.amazon.com/){.external} is the only supported cloud provider.
+    * [Bamboo DC](https://confluence.atlassian.com/bamboo/bamboo-8-1-release-notes-1103070461.html){.external} is the only supported DC product
+
+    Support for additional Cloud providers and DC products will be made available in future.
 
 1. Check if Terraform is already installed by running the following command:
    ```shell
    terraform version
    ```
-2. If Terraform is not installed, install it by following the [official instructions](https://learn.hashicorp.com/tutorials/terraform/install-cli).
+2. If Terraform is not installed, install it by following the [official instructions](https://learn.hashicorp.com/tutorials/terraform/install-cli){.external}.
 
 ### :material-package: Helm
 
-Atlassian supports Helm Charts for some of its [Data Center products](https://atlassian.github.io/data-center-helm-charts/), including Bamboo. This project uses Helm charts to package Bamboo Data Center as a turnkey solution for your cloud infrastructure.
+Atlassian supports Helm Charts for some of its [Data Center products](https://atlassian.github.io/data-center-helm-charts/){.external}, including Bamboo. This project uses Helm charts to package Bamboo Data Center as a turnkey solution for your cloud infrastructure.
```

Comment:
```suggestion
Atlassian supports Helm Charts for its [Data Center products](https://atlassian.github.io/data-center-helm-charts/){.external}, including Bamboo. This project uses Helm charts to package Bamboo Data Center as a turnkey solution for your cloud infrastructure.
```

Reply:
Currently we only support Bamboo deployments, so is making this change not giving the impression that we support all DC products? Following on, if we want to commit this recommendation then we should probably remove the words `including Bamboo.`?